packages/assets/scss/03-organisms/_page-flipper.scss

```diff
@@ -35,6 +35,7 @@
       align-items: stretch;
     overflow: hidden;
     width: 100%;
+    list-style: none;
```

Comment:
seeing some visual regressions to the listing styles. I think the styles might need to go into lower components
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22292_binder-page-a11y-improvements/?p=molecules-arrow-nav
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22292_binder-page-a11y-improvements/?p=molecules-arrow-nav-as-next-page

![Screen Shot 2021-07-01 at 6 42 29 PM](https://user-images.githubusercontent.com/5789411/124197482-9ab15b80-da9c-11eb-9984-dff35f91ba11.png)
![Screen Shot 2021-07-01 at 6 42 33 PM](https://user-images.githubusercontent.com/5789411/124197483-9ab15b80-da9c-11eb-8dea-32ebce687dc7.png)